### ISSUE TITLE
How can I run multiple files at once?

### ISSUE THREAD

**Issue by uhlhosting** [SalOne22/rimage#206]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Hi,

trying to run:

```
rimage webp *.png -f webp -r
```

I get: 

```
> rimage webp *.png -f webp -r
error: invalid value 'webp' for '--codec <CODEC>': webp is not supported codec.

For more information, try '--help'.
```

Any ideas?

### Library Version

0.11.0 (latest)

**Comment by SalOne22** [comment]
Could you please give me the version that you used? Run `rimage --version`. We have breaking changes in latest version, so some options will not work.

**Comment by uhlhosting** [comment]
@SalOne22 hi

> rimage --version
rimage 0.10.3


seems latest was not deploying.

**Comment by SalOne22** [comment]
> seems latest was not deploying.

Yup, latest version is pre release. You can find pre releases in releases tab. Version v0.11.0 will be released after icc issue resolved.

**Comment by SalOne22** [comment]
Added warning to the readme file about pre-releases

**Comment by Hahlh** [comment]
I ran into the same issue.
It would be great if you could provide a newer version via crates.io.

**Comment by SalOne22** [comment]
I released version 0.11 for the public, so there should be no similar issues. You can install it with cargo or download prebuild binaries from [releases tab](https://github.com/SalOne22/rimage/releases/tag/v0.11.0)
